src/Controls/src/Xaml/XmlTypeXamlExtensions.cs

```diff
@@ -61,9 +61,9 @@ static class XmlTypeXamlExtensions
 			}
 
 			var lookupNames = new List<string>();
```

Comment:
While we're looking at this, do we know an estimate of the `List<string>` capacity? `lookupName.Count + 2`?

Reply:
There will only ever be 1 or 2 items in this list, so I guess I can set the initial capacity to 2.